ckan/templates-midnight-blue/package/search.html

```diff
@@ -62,11 +63,12 @@
 
 {% block secondary_content %}
   <div class="filters">
+    <span class="close no-text hide-filters" tabindex="0"><i class="fa-solid fa-xmark"></i><span class="text">close</span></span>
     <div>
+      <h2 class="d-sm-block d-md-none">Filter results</h2>
```

Comment:
All user facing strings in templates need to be translated (i.e wrapped in `{{ _("...") }}`)
```suggestion
      <h2 class="d-sm-block d-md-none">{{ _("Filter results") }}</h2>
```